npm/mount-utils/src/index.ts

```diff
@@ -220,6 +31,38 @@ export function setupHooks (optionalCallback?: Function) {
   // @ts-ignore
   Cypress.on('test:before:run', () => {
     optionalCallback?.()
-    cleanupStyles()
   })
 }
+
+/**
+ * Remove any style or extra link elements from the iframe placeholder
+ * left from any previous test
+ *
+ * @deprecated Removed as of Cypress 11.0.0
+ */
+export function cleanupStyles () {
+  throw new Error('cleanupStyles has been removed')
```

Comment:
This is pretty jarring - I think we need either

- deprecate but don't error - just warn?
- or something like "`cleanupStyles` is deprecated. Consider <link to useful migration docs with recommendations".

Reply:
https://github.com/cypress-io/cypress/pull/24419#issuecomment-1298048924